cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -166,91 +166,94 @@ <h2 class="h4">
                               {{ zipcode | escape }}
                             </h2>
                         </div>
-                        <table class="o-table o-table__stack-on-small u-w100pct">
+
+                        <table class="o-table o-table__stack-on-small">
                             <thead>
                                 <tr>
-                                    <th scope="col">
-                                        <div class="rich-text">
-                                            <p>Agency</p>
-                                        </div>
+                                    <th>
+                                        <p>Agency</p>
```

Comment:
These headers should not have `<p>` tags.

Reply:
Removed